# Request
Header header
object_recognition_msgs/Table floor
visualization_msgs/MarkerArray obstacles
string  policy
bool    print_evaluation
---
# Response
geometry_msgs/PoseStamped   set_pose
float32 nn_distance
float32 hull_distance